fonttools/genft.cpp

```diff
@@ -81,6 +81,7 @@ QMap<QString, QString> nmap {
       { "rests.5",                  "rest32nd" },
       { "rests.6",                  "rest64th" },
       { "rests.7",                  "rest128th" },
+      { "rests.8",                  "rest256th" },
```

Comment:
Good point - my understanding is this code is intended to create links between Lilypond's Feta/Emmentaler font and the SMuFL compliant Mscore font. Glyphs for `rests.9` and `rests.10` are not defined in Feta. Actually now that I double check [http://lilypond.org/doc/v2.18/Documentation/notation/the-feta-font#rest-glyphs](url) it looks like `rests.8` isn't defined either, so this might be leftover from my testing and I should just cut that line too. Or am I misunderstanding the purpose of this code?